indico/queries/submission.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import Dict, List
+
+from indico.client.request import GraphQLRequest
+from indico.types import Job, Submission
+
+
+class ListSubmissions(GraphQLRequest):
```

Comment:
I think this could really use a docstring - esp on how to use the filters.
I'd also make sure a default `limit` is supplied. iirc, if limit is not provided, filters are ignored. so we can have a default of 1000 or something

Reply:
or maybe default limit if filters is supplied and limit isn't? or maybe we do that in Fog.... or maybe we fix atmos so folks can filter without limit :thinking: idk, what do you think?